### ISSUE TITLE
Remove redundant and unused #Attributes

### ISSUE THREAD

**Issue by marwin89** [Vocaluxe/Vocaluxe#626]
## Suggestion:
Remove duplicated or outdated #Attributes

- #DUETSINGERP1/P2
- #RESOLUTION 
- #NOTESGAP
- #ENCODING 
- #RELATIVE
- #ALBUM
- #SOURCE 
- #YOUTUBE
- #LENGTH
- #PREVIEW

(according to [official common Song format Specification](https://usdx.eu/format/) v1.0.0 - release on: 1th Dec 2023)

**Comment by Stefan1200-de** [comment]
Since this attributes are still optional, where is the problem in keeping them?

Link to Song Loader parsing these attributes: 
https://github.com/Vocaluxe/Vocaluxe/blob/dbc1738da1c848facb7be6b5596ebad8b0ba9f44/VocaluxeLib/Songs/CSongLoader.cs#L133

**Comment by flokuep** [comment]
Yeah, I'd also suggest to keep them as they are - at least until there is more active vocaluxe development.

**Comment by Stefan1200-de** [comment]
> Yeah, I'd also suggest to keep them as they are - at least until there is more active vocaluxe development.

Yes, but even if the development would be more active, some attributes should still be read, as long as this attributes are not needed in the new song format specification, to be compatible with old txt files. Especially DUETSINGERP1/2 is used by many txt files in my collection. But that is just my opinion.

**Comment by flokuep** [comment]
True, but if the community and everything around karaoke games comes to the point, that versioned song-files are default and there is some tooling to do the migrations, Vocaluxe could start with a slow deprecation of these tags - To keep the code base clean and remove unncecessary code.
